tailwind.config.js

```diff
@@ -6,10 +6,15 @@ module.exports = {
   ],
   theme: {
     extend: {
+      colors: {
+        purple: "#5602E0",
+        "dark-gray": "#9A9A9A",
+        "light-gray": "#E4E4E4",
```

Comment:
It could be better if you write this color in consistent way like all in quotes or no quotes at all

Reply:
I tried it but then when I save they automatically get removed, to mean it is a correction itself.